test/test_vocab.py

```diff
@@ -135,7 +135,7 @@ def test_vocab_download_fasttext_vectors(self):
     def test_vocab_extend(self):
         c = Counter({'hello': 4, 'world': 3, 'ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T': 5, 'freq_too_low': 2})
         # Build a vocab and get vectors twice to test caching.
-        for i in range(2):
+        for _ in range(2):
```

Comment:
Naming a loop variable and not using it, is a common lint issue. Using ```_``` prevents this.